gallery/src/PageHeader/PageHeader.scss

```diff
@@ -0,0 +1,53 @@
+.nx-page-header {
```

Comment:
The gallery's header should have both the `.nx-page-header` class and its own gallery-specific class. The gallery stylesheets should only style the latter, and only add the minimum necessary in addition to the `.nx-page-header` styles.  For instance I wouldn't expect to see the `grid-area` being repeated here.